assets/tasks/common.json

```diff
@@ -236,10 +238,12 @@
                 "env": {
                     "DOCKER_PSSWD": "${command:docker_password}",
                     "DOCKER_LOGIN": "${command:docker_login}",
-                    "VSCODE_CMD": "--verbose platform push --credentials credentials.zip --package-name __change__ --package-version ${command:tcb.getNextPackageVersion} --login $DOCKER_LOGIN $DOCKER_PSSWD --canonicalize docker-compose.prod.yml"
+                    "DOCKER_REGISTRY": "${command:docker_registry}",
+                    "VSCODE_CMD": "--verbose platform push --credentials credentials.zip --package-name __change__ --package-version ${command:tcb.getNextPackageVersion} --login-to $DOCKER_REGISTRY $DOCKER_LOGIN $DOCKER_PSSWD --canonicalize docker-compose.prod.yml"
                 }
             },
             "args": [
+                "DOCKER_REGISTRY=${DOCKER_REGISTRY:-registry-1.docker.io}",
```

Comment:
is this right?

Reply:
Yes, this one is needed because the DockerHub registry can be empty, nut when using it on the `tcb platform push` the command was done in a way that it cannot. So what this does is attribute the `registry-1.docker.io` value (default value for Dockerhub registry) to the `DOCKER_REGISTRY` variable if it is empty

Reply:
@andreriesco Also I think that this behavior is not implemented in the `tasks.ps1` interpreter. So, would be good to have another solution for this.

Reply:
It is a bash command, but I moved this check to the `tcb-env-setup.sh` script to make it sure the task works